andela/ah-frontend-odin
# #161340137 Enable users to signup through the UI

#### What does this PR do?
Users should be able Signup/Register to Authors Haven

#### Description of Task to be completed?
- create the user signup form
- create user signup react components
- write tests for the user signup react components

#### How should this be manually tested?
-  pull branch `ft-user-signup-161340137)` and check it out 
- Install the package dependencies by running `yarn` or `npm install`
- start server by running `yarn start`
- visit  this URL: `http://localhost:3000/register`  and start the manual testing
- you can also run tests by running `yarn  test ` or `yarn run test:coverage ` command

#### What are the relevant pivotal tracker stories?
[#161340137](https://www.pivotaltracker.com/story/show/161340137)
#### Screenshots
<img width="1680" alt="screen shot 2018-12-05 at 21 40 08" src="https://user-images.githubusercontent.com/39129473/49535936-6ff7ca00-f8d6-11e8-8f88-9818daa8b90a.png">

- How an error message pops up
<img width="1680" alt="screen shot 2018-12-06 at 12 30 46" src="https://user-images.githubusercontent.com/39129473/49575068-dd494080-f952-11e8-9932-181a738eb00b.png">
